src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileFromFileName = new File(fromFileName);
+        File fileToFileName = new File(toFileName);
+        Map<String, Integer> result = new HashMap<>();
+
+        if (!fileFromFileName.exists() || !fileToFileName.exists()) {
+            return;
+        }
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
Use try-with-resources with FileReader as well to ensure that all resources are closed properly.